src/components/inputs/money-input/money-input.js

```diff
@@ -149,23 +149,42 @@ const createCurrencySelectStyles = ({
 //  "fractionDigits": 7
 // }
 // which equals 0.0123456 €
-
-// Parsing
-// Since most users are not careful about how they enter values, we will parse
-// both `.` and `,` as decimal separators.
+// when locale is provided we use the number 9999.999 becauause it'll either be formated to
+// A: 9,999.999
+// B: 9.999,999
+// otherwise, we infere the throwaway, separator from the amount
 // When a value is `1.000,00` we parse it as `1000`.
 // When a value is `1,000.00` we also parse it as `1000`.
-//
+// Since most users are not careful about how they enter values, we will parse
+// both `.` and `,` as decimal separators.
 // This means the highest amount always wins. We do this by comparing the last
 // position of `.` and `,`. Whatever occurs later is used as the decimal
 // separator.
-export const parseRawAmountToNumber = rawAmount => {
+export const getLocalesSperatorAndThrowaway = (rawAmount, locale) => {
```

Comment:
What is a `throwaway`? 🤔

Reply:
the thousands separator symbol

Reply:
Can we find a better name? I find it confusing to understand what it does/is by the name